generator/transformers/common_producer.py

```diff
@@ -156,9 +156,7 @@ def extract_description(data, length: int = 2048) -> list:
         """
         if not data:
             return []
-        if isinstance(data, list):
-            data = ' '.join(data)
-        return textwrap.wrap(re.sub(r'(\s{2,}|\n|\[@TODO.+)', ' ', data).strip(), length)
```

Comment:
I think we still need to keep the `re.sub` method because it cleans up extra whitespace and newlines. `\s{2,}` strips extra whitespace, and `\n` strips newlines. I think we can just remove `\[@TODO.+`.

For example in the `GPSData` struct, the `shifted` parameter description has multiple lines of text separated by a newline. This PR is generating documentation with a lot of extra spaces: 

```
/**
 * 
                True, if GPS lat/long, time, and altitude have been purposefully shifted (requires a proprietary algorithm to unshift).
                False, if the GPS data is raw and un-shifted.
                If not provided, then value is assumed False.
            
 *
 * @since SDL 6.0.0
 */
```

When we want: 
```
/**
 * True, if GPS lat/long, time, and altitude have been purposefully shifted (requires a proprietary algorithm to unshift). False, if the GPS data is raw and un-shifted. If not provided, then value is assumed False.
 *
 * @since SDL 6.0.0
 */
```